src/ai/backend/manager/models/gql_models/agent.py

```diff
@@ -722,11 +722,14 @@ async def batch_load(
         graph_ctx: GraphQueryContext,
         agent_ids: Sequence[AgentId],
         *,
-        domain_name: str | None,
+        access_key: Optional[str] = None,
```

Comment:
```suggestion
        access_key: AccessKey,
```
Now it passes the type check without `Optional` annotation